Scripts/fn_forceShooting.sqf

```diff
@@ -0,0 +1,65 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Jonpas, Rory, Tyrone
+ * Orders a unit to target an object and begin firing until either the shooter or target is dead or the timer elapses.
+ * Firemode types Single (0) and Full auto (1)
+ * Set firing timer to 0.1 for full auto
+ * Designed ONLY for infantry, vehicle results not guaranteed.
+ *
+ * Arguments:
+ * 0: AI Shooter <OBJECT>
+ * 1: Target <OBJECT>
+ * 2: Firemode <NUMBER> (default: 0)
+ * 3: Timer <NUMBER> (default: 10)
+ * 4: Time between shots <NUMBER> (default: 1)
```

Comment:
Is `0.1` the lowest it should ever be set? What happens if you set it to `0`?

Reply:
Nothing really, just figured for no difference it's probably better to not run it on every frame.

Reply:
You can limit it then.
```
_firingTimer = _firingTimer max 0.1
```